src/test/java/org/apache/commons/lang3/time/CalendarUtilsTest.java

```diff
@@ -83,4 +87,12 @@ public void testGetYear() {
         assertEquals(Calendar.getInstance().get(Calendar.YEAR), CalendarUtils.INSTANCE.getYear());
     }
 
+    @Test
+    public void testToLocalDate() {
+        Calendar calendar = new GregorianCalendar(TimeZone.getTimeZone("GMT"));
```

Comment:
FYI: You can reuse `org.apache.commons.lang3.time.TimeZones.GMT` here.

Reply:
fixed